libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -52,12 +71,48 @@ getDState :: NewEpochState era -> DState era
 getDState = certDState . lsCertState . esLState . nesEs
 
 queryConstitution :: EraGov era => NewEpochState era -> Maybe (Constitution era)
-queryConstitution nes =
-  getConstitution (nes ^. nesEpochStateL . esLStateL . lsUTxOStateL . utxosGovStateL)
+queryConstitution = getConstitution . queryGovState
 
 queryConstitutionHash ::
   EraGov era =>
   NewEpochState era ->
   Maybe (SafeHash (EraCrypto era) AnchorData)
 queryConstitutionHash nes =
   anchorDataHash . constitutionAnchor <$> queryConstitution nes
+
+-- | This query returns all of the state related to governance
+queryGovState :: NewEpochState era -> GovState era
+queryGovState nes = nes ^. nesEpochStateL . esLStateL . lsUTxOStateL . utxosGovStateL
+
+-- | Query DRep state.
+queryDRepState ::
+  NewEpochState era ->
+  -- | Specify state of which DReps should be returned. When this set is empty,
```

Comment:
Just a nitpick, i find the phrasing a little confusing (took me a few readings)
Maybe:  `Specify Dreps whose state should be returned` ?

Reply:
Thanks. Done